README.md

```diff
@@ -23,42 +18,24 @@ kind load docker-image redis-demo-worker --name={cluster$i}
 5. go (for installation, configuration and basic testing, follow instructions in e.g. [How to Install GoLang (Go Programming Language) in Linux](HTtps://www.tecmint.com/install-go-in-linux/); make sure go environment variables and path are configured accordingly)
 
 ## Environment Preparation
-1. Make sure there exist a kubernetes namespace with name edge-namespace with Istio sidecar injection enabled
-2. Make sure the application-generator folder is located under path ~/go_projects/src/ and initialize module by executing go mod init
-3. If needed, install go module dependencies, e.g. cobra and yaml
-4. Deploy InfluxDB by running startup.sh script and providing no of clusters as argument
-    ```bash
-    ./startup.sh {$no_of_clusters}
-    ```
-5. Modify any of the service chain files under the chain directory according to the requirements.
-6. Modify any of the cluster placement files under the clusters directory according to the requirements.
-7. Generate and deploy kubernetes manifest files by running 'generator.sh' script. It accepts three arguments, path to chain file, path to cluster file and value for readiness probe in seconds.
+1. Make sure the application-generator folder is located under path ~/go_projects/src/ and initialize module by executing go mod init
+2. If needed, install go module dependencies, e.g. cobra and yaml
+3. Modify any of the input files under the input directory according to the requirements.
+4. Generate and deploy kubernetes manifest files by running 'generator.sh' script. It accepts three arguments, path to chain file, path to cluster file and value for readiness probe in seconds.
```

Comment:
now generator.sh only accepts two arguments instead...

Reply:
I didn't see that part

Reply:
It is now fixed, just checked the commit with new_hash.